osmhelper/__init__.py

```diff
@@ -286,7 +286,7 @@ def refresh_route(route_ref, route_type, bbox):
         <bbox-query e="%s" n="%s" s="%s" w="%s"/>
     </query>
     <print/>
-    """ % (route_type, str(route_ref), bbox["e"], bbox["n"], bbox["s"], bbox["w"]))
+    """ % (route_type, str(route_ref), str(bbox["e"]), str(bbox["n"]), str(bbox["s"]), str(bbox["w"])))
```

Comment:
Because we are not using Python 3 and json data structures are imported and stated as unicode. Without those it just doesn't work :)